redfish-core/lib/oem/ibm/usb_code_update.hpp

```diff
@@ -63,8 +63,6 @@ inline void getServiceName(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
                 return;
             }
 
-            BMCWEB_LOG_ERROR << "Can't find usb code update service!";

```

Comment:
Do we still want the debug?

Reply:
Done. Yes, I think we want the debug. I have added it.